<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>훈련시키기</string>
  </property>
  <widget class="QCheckBox" name="knn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>KNN(K-Nearest Neighbors)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="dtc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DecisionTreeClassifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="train">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>훈련시키기</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetMod">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>아직 훈련된 모델이 없습니다</string>
   </property>
  </widget>
  <widget class="QLabel" name="accLab">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>정확도: 0%</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>22</width>
     <height>191</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>140</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>훈련에 사용할 데이터: %개</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelte">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>테스트에 사용할 데이터: %개</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LogisticRegression</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="br">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BayesianRegression</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mlp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MLPClassifier</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="xBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="yBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="targetMod_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X축</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetMod_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y축</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
